### DISCUSSION TITLE
UI 디자인 Asset

### DISCUSSION THREAD

**Discussion by KimDa99** [SeoulTechTCPGame/block-it#30] [Show and tell]
Png는 레스터 이미지, SVG는 벡터 이미지입니다.

png파일의 해상도가 적절치 못하다고 생각하신다면,
unity에는 기본적으로 svg를 넣지 못하니, svg를 적절한 배율 (픽셀배율)을 설정하여 원하는 해상도로 다시 뽑아내시길 바랍니다

만일 svg를 unity에 그대로 쓰고 싶으시다면, 추가 플러그인을 찾아봐야합니다. 

# PlayScene
[Quoridor_PlayScene_png.zip](https://github.com/junnie082/block-it/files/11752649/Quoridor_PlayScene_png.zip)
[Quoridor_PlayScene_svg.zip](https://github.com/junnie082/block-it/files/11752650/Quoridor_PlayScene_svg.zip)

**Comment by KimDa99** [comment]
# Option 
- 볼륨 아이콘
[volumn_ver1_svg.zip](https://github.com/junnie082/block-it/files/11753839/volumn_ver1_svg.zip)
[volumn_ver1_png.zip](https://github.com/junnie082/block-it/files/11753840/volumn_ver1_png.zip)

staled
[Vol_png.zip](https://github.com/junnie082/block-it/files/11752734/Vol_png.zip)
[Vol_svg.zip](https://github.com/junnie082/block-it/files/11752735/Vol_svg.zip)

- 토글
[toggle_svg.zip](https://github.com/junnie082/block-it/files/11752738/toggle_svg.zip)
[toggle_png.zip](https://github.com/junnie082/block-it/files/11752739/toggle_png.zip)

**Comment by gomdoricake** [comment]
# Tutorials

[Tutorials.zip](https://github.com/junnie082/block-it/files/11752934/Tutorials.zip)
[NewTotorials.zip](https://github.com/junnie082/block-it/files/11795399/blockitExport.zip)
[pawns.zip](https://github.com/junnie082/block-it/files/11795751/pawns.zip)

**Comment by gomdoricake** [comment]
# Loading

[Loading.zip](https://github.com/junnie082/block-it/files/11796609/Loading.zip)

**Comment by gomdoricake** [comment]
# 한글 폰트(나눔 스퀘어 라운드)

[nanum-square-round.zip](https://github.com/SeoulTechTCPGame/block-it/files/12024002/nanum-square-round.zip)

**Reply by seong0929** [reply]
[Block It language](https://docs.google.com/spreadsheets/d/1Dsj19n_rK5MEaxfu_4dn2NMJHAj3pcd4A-eY-7MjJ2M/edit#gid=0) 스프라이트 시트로 언어 번역할 예정입니다. 이외에 또 필요한 문구를 정리해서 전달해주세요

**Reply by gomdoricake** [reply]
https://hangeul.naver.com/font/nanum

**Comment by gomdoricake** [comment]
# 로딩 이미지

![loading-2 1](https://github.com/SeoulTechTCPGame/block-it/assets/88719152/76a8897f-29d3-440f-9a42-cba8fbe02126)
